examples/notebooks/7_Data_Base.ipynb

```diff
@@ -0,0 +1,939 @@
+{
```

Comment:
I would name this file differently: the goal is not to have a DB per say, but to be able to save and load results. I have to checkout this file, but this could even be part of another notebook

Reply:
I stand by that, with the `Result` integration, I would even have just a small mention in one of the first notebooks